### PR TITLE
css-brochure-form - Tré Mitchell

### PR DESCRIPTION
**Pull request by TreMitchell**
Screenshot 2024-09-05 at 12 54 07 PM

### CODE REVIEW

File: css-brochure-form/index.html [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  <body>
+    <div class="brochure">
+      <h2>Learn More</h2>
+      <form>
+        <label for="email">Company Email</label>
+        <div>
+          <input
+            type="email"
+            name="company-email"
+            placeholder="sally@company.co" />
```
Comment (shawnkost) [inline css-brochure-form/index.html:22]: Missing an `id="email"` to match the `for` of the `label`

File: css-brochure-form/index.html [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+        </div>
+        <label for="name">Name</label>
+        <div>
+          <input name="name" placeholder="Sally Smith" />
```
Comment (shawnkost) [inline css-brochure-form/index.html]: Missing an `id="name"` to match the `for` of the `label`

File: css-brochure-form/index.html [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+        </div>
+        <label for="location">Where are you hiring?</label>
+        <div>
+          <input name="location" placeholder="City, Country" />
```
Comment (shawnkost) [inline css-brochure-form/index.html]: Missing an `id="location"` to match the `for` of the `label`

File: css-brochure-form/styles.css [hunk 4]
```diff
@@ -0,0 +1,57 @@
+* {
+  font-family: 'Brandon Grotesque', sans-serif;
+}
+
+h2 {
+  text-transform: uppercase;
+  color: #ec5c57;
+  justify-content: center;
```
Comment (shawnkost) [inline css-brochure-form/styles.css:9]: If the `h2` doesn't have `display: flex`, this `justify-content` isn't doing anything

**Review by shawnkost — CHANGES_REQUESTED** [review]
Solid first attempt! Some changes here

**Comment by TreMitchell** [comment]
Added the IDs of name, email, and location while making the display of header 2 become flexed. The corrected code is in the other commit.

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**